palette.js

```diff
@@ -1,110 +1,105 @@
-class View{
-    constructor(body){
-        this.body = body;
-    }
-    on_submit(cb){
-        this.body.onkeypress = cb;
-    }
-    on_load(cb){
-        this.body.onload = cb
-        console.log("loaded")
-    }
+class View {
+  constructor(body) {
+    this.body = body;
+  }
+  on_submit(cb) {
```

Comment:
I think you should have a consistent naming convention for functions across these various classes, either `snake_case` or `camelCase`. I'd suggest the latter like you have in the `Color` class.
```suggestion
  onSubmit(cb) {
```